tests/stubtest_allowlists/py37.txt

```diff
@@ -3,6 +3,8 @@ _collections_abc.AsyncGenerator.ag_code
 _collections_abc.AsyncGenerator.ag_frame
 _collections_abc.AsyncGenerator.ag_running
 _dummy_threading
+asyncio.AbstractEventLoop.run_in_executor
+asyncio.events.AbstractEventLoop.run_in_executor
```

Comment:
This is a true positive -- it's `async` in the 3.7 branch of CPython; it was change to be a synchronous method in 3.8. https://github.com/python/cpython/blob/811f65ba263140b6ba28151246b52efe149a6382/Lib/asyncio/events.py#L289

Reply:
I think we should still allowlist it. If you've implemented the AbstractEventLoop interface, you probably don't want some type checker yelling at you for returning a Future (if you check wit 3.7).

Reply:
Okay, fair enough!

Reply:
Maybe we could add a comment to that effect?